### PR TITLE
Create file_utils.py

### PR DESCRIPTION
**Pull request by MonarchChakri**
Utility file helpful for file I/O in python3.
Helps for plain text IO and JSON load and dump.

### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.

### CODE REVIEW

**Comment by MonarchChakri** [comment]
Hi @cclauss / @poyea
Added file_utils.py under others and created this PR for https://github.com/TheAlgorithms/Python/issues/2510
A utility files helpful for all common file IO.
Please review and merge.

File: other/file_utils.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        sys.stdout.flush()
+        print("Unable to create file {}".format(file_name))
+        sys.stdout.flush()
+
+
+def dump_json_to_file(data, file_name):
+    touch_file(file_name)
+    with open(file_name, "w") as write_file:
+        write_file.write(json.dumps(data, indent=4))
+        write_file.write("\n")
```
Comment (cclauss) [inline other/file_utils.py:40]:
```suggestion
        write_file.write(json.dumps(data, indent=4) + "\n")
```

File: other/file_utils.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+
+def load_json_from_file(file_name):
+    touch_file(file_name)
+    with open(file_name) as readfile:
+        content = readfile.read()
+    if content == "":
+        content = str(dict())
+    content = json.loads(content)
+    return content
```
Comment (cclauss) [inline other/file_utils.py:50]:
```suggestion
        return readfile.read() or str({})
```

File: other/file_utils.py [hunk 3]
```diff
@@ -0,0 +1,129 @@
+import json
+import sys
+
+
+def touch_file(file_name):
```
Comment (cclauss) [inline other/file_utils.py:5]: Type hints?  Doctests?  Algorithmic functions should not print().  See CONTRIBUTING.md.

**Comment by MonarchChakri** [comment]
@cclauss 
Will make the changes and update the file

**Comment by MonarchChakri** [comment]
@cclauss
Updated the file, please review

File: other/file_utils.py (hunk 3 above)
Comment (MonarchChakri) [reply other/file_utils.py]: Changes done and sent for review

**Comment by MonarchChakri** [comment]
@poyea 
Can you also please approve and merge. 😅

**Review by 53jk1 — APPROVED** [review]

**Pull request closed without merging**